src/components/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,54 @@
+import { Box, Button, HStack, Image, Text } from '@chakra-ui/react';
+
+import { publicRuntimeConfig } from '../../../next.config';
+
+import { NavigationLink } from './NavigationLink';
+
+const navigations = [
+  {
+    name: 'about us',
+    href: '/',
+  },
+  {
+    name: 'features',
+    href: '/',
+  },
+  {
+    name: 'status',
+    href: '/',
+  },
+  {
+    name: 'discover room',
+    href: '/',
+  },
+];
+
+function Navigation() {
+  // TODO: add mobile navigation
+  return (
+    <Box background='#181818' py='2' px='2'>
+      <HStack justifyContent='space-between'>
+        <HStack>
+          <Image src='/img/logo-bnw.svg' alt='Mocha Logo' />
+          <Text variant='regular' color='white'>
+            mocha
+          </Text>
+        </HStack>
+        <HStack spacing='8'>
+          {navigations.map((navigation) => (
+            <NavigationLink key={navigation.name} {...navigation} />
+          ))}
+        </HStack>
+        <Button
+          as='a'
+          variant='primary'
+          textColor='white'
+          href={publicRuntimeConfig.botInvitationUrl}>
```

Comment:
please move this url to props, so that this component can be reused for other purpose and this component doesnt need to know from where the data is

sorry for the lack of documentation, but this project is going to use this pattern https://www.patterns.dev/react/presentational-container-pattern/

TLDR: 
![image](https://github.com/mocha-bot/land/assets/47074517/12308dd4-2c31-4629-83c3-8417d01d1d9f)